gammapy/background/tests/test_reflected.py

```diff
@@ -62,29 +63,39 @@ def test_find_reflected_regions(exclusion_mask, on_region):
     )
     fregions.run()
     regions = fregions.reflected_regions
-    assert len(regions) == 14
-    assert_quantity_allclose(regions[3].center.icrs.ra, Angle("83.674 deg"), rtol=1e-2)
+    assert len(regions) == 15
+    assert_quantity_allclose(regions[3].center.icrs.ra, Angle("82.592 deg"), rtol=1e-2)
 
     # Test without exclusion
     fregions.exclusion_mask = None
     fregions.run()
     regions = fregions.reflected_regions
-    assert len(regions) == 16
+    assert len(regions) == 17
 
     # Test with too small exclusion
-    small_mask = exclusion_mask.cutout(pointing, Angle("0.2 deg"))
+    small_mask = exclusion_mask.cutout(pointing, Angle("0.1 deg"))
     fregions.exclusion_mask = small_mask
     fregions.run()
     regions = fregions.reflected_regions
-    assert len(regions) == 16
-    assert_quantity_allclose(regions[3].center.icrs.ra, Angle("83.674 deg"), rtol=1e-2)
+    assert len(regions) == 17
+    assert_quantity_allclose(regions[14].center.icrs.ra, Angle("83.868 deg"), rtol=1e-2)
```

Comment:
This is due to the change of rotation angle in the region definition?

Reply:
yes, I think so